packages/e2e-test-utils/src/click-on-more-menu-item.js

```diff
@@ -14,7 +14,7 @@ import { toggleMoreMenu } from './toggle-more-menu';
 
 const SELECTORS = {
 	postEditorMenuContainer:
-		'//*[contains(concat(" ", @class, " "), " edit-post-more-menu__content ")]',
+		'//*[contains(concat(" ", @class, " "), " interface-more-menu-dropdown__content ")]',
```

Comment:
Can we target using something that's user oriented e.g. ARIA label?

Reply:
Could be tricky as the block toolbar has an identical label for its settings menu.

I would like to get rid of this logic that uses a different class name for the site editor:
https://github.com/WordPress/gutenberg/blob/11ae04ca6957c85e99f2091692104f003f24323f/packages/e2e-test-utils/src/click-on-more-menu-item.js#L15-L20

I'll be able to remove that once I've done the migration for the site editor, so at that point I'll loop back and improve this.

Reply:
In my experience user-facing labels change more often than classnames 😅 
It's always a compromise with e2e selectors; it's not a huge deal if we have to change them once in a while. Classnames also tend to be more unique than aria-labels.